examples/crowdfunding/test/happypaths.js

```diff
@@ -101,9 +101,14 @@ describe('Crowdfunding Tests - Happy Paths', function () {
       convert.uint64ToBigEndian(fundCloseTs)
     ];
 
-    // create application
-    applicationId = runtime.addApp(
-      { ...creationFlags, appArgs: creationArgs }, {}, approvalProgram, clearProgram).appID;
+    // deploy application
+    applicationId = runtime.deployApp(
+      approvalProgram,
+      clearProgram,
+      { ...creationFlags, appArgs: creationArgs },
+      {}
```

Comment:
i am not sure if this is how we pass params in `deployer.deployApp`. Could you please confirm?

Reply:
When i check the deploy.deployApp, this is list of parammeters 
```
async deployApp (
    approvalProgram: string,
    clearProgram: string,
    flags: rtypes.AppDeploymentFlags,
    payFlags: wtypes.TxParams,
    scTmplParams?: SCParams,
    appName?: string)
```
and I just update runtime.deployApp() follow this.

Reply:
difference: `approvalProgram` & `clearProgram` are file names here, and in `runtime.deployApp` it's the TEAL code

Reply:
so we probably need to add `scTmplParams?` as well

Reply:
Can you give me a short description about scTmplParams?

Reply:
@vuvth you have it in the parameters list: `scTmplParams?: SCParams,`

Reply:
Yes, i added it to runtime.deployApp.